source/includes/fact-query-converter-generic.rst

```diff
@@ -1,15 +1,16 @@
 - The query converter uses AI technology which may not be able to 
-  convert long or complex queries, triggers, or stored procedures. 
-  Some queries may not be converted correctly while others may not 
-  be converted at all.
+  convert long or complex queries, triggers, packages, or stored 
+  procedures. Some queries may not be converted correctly while 
+  others may not be converted at all.
 
 - The query converter uses the relational schema, the MongoDB schema,  
   and the mapping rules in your current project to determine how the 
   queries should be converted. Conversions may fail or be incorrect if 
   the queries reference tables that are not in your relational schema
   or if they are not mapped to MongoDB collections.
 
-- Converted queries, triggers, views, and stored procedures are saved in 
-  your project and persist through project import and exports.
+- Converted queries, triggers, views, packages, and stored procedures 
+  are saved in your project and persist through project import and 
+  exports.
```

Comment:
(suggestion) 
It might be good to add [this link](https://www.mongodb.com/docs/generative-ai-faq/), somewhere on the page.

Reply:
Rather than linking to the FAQ, I added a link to the Query Converter `AI & Data Usage` page as it is more context specific and it also links to the FAQ in the second sentence.